lib/SVN/Notify.pm

```diff
@@ -1471,10 +1471,10 @@ sub output_headers {
     # Q-Encode the phrase part of recipient headers.
     my $norm;
     if (PERL58) {
-        require Email::Address;
+        require Email::Address::XS;
```

Comment:
Hi! No need to put `Email::Address::XS` into `if(PERL58)` block. Module `Email::Address::XS` is compatible and tested back to Perl version 5.6.0.